### DISCUSSION TITLE
How to plot PDF yaxis out of plane?

### DISCUSSION THREAD

**Discussion by matheusGariglio** [mtex-toolbox/mtex#1238] [Ask Anything]
Hi,

I would like to know how can I plot the yaxis of my PDF out of plane?

Right now I have the plot like this (yaxis = DC):

![image](https://user-images.githubusercontent.com/77332633/135923587-c077da17-75ea-4053-bd74-530178992c45.png)

But I would like it to be like this :

![image](https://user-images.githubusercontent.com/77332633/135923975-441f434f-52f4-46fd-a429-80828590c1d9.png)

Cheers,

Matheus

**Comment by kilir** [comment]
Hi,
at the moment the x,y directions will always be displayed in plane. You your purpose, you can rotate DC about 90 degree around DR.
Cheers,
Rüdiger

**Reply by matheusGariglio** [reply]
Hi Rüdiger,

Thanks for your answer! However, when I do...

`ebsd2 = rotate(ebsd2,rotation.byAxisAngle(xvector,90*degree));`

...in order to rotate DC about 90° around DR, the plot axis continue to show X on the right and Y on the bottom. Shouldn't it be changing the plot axis to X on the right and Z on the bottom/top, as so the Y axis is in/out of plane?

Cheers,

Matheus

**Reply by kilir** [reply]
Hi,
no, it should not since you rote your orientations within your specimen reference frame (in your case labeled x,y,z but could be any string) which will always print with the specimen directions 100 and 010 in the plane of the screen. 
If your rotate your orientations, you can also consider rotating your labels when displaying something in specimen coordinates (e.g. a PDF).

Hope that helps.

Cheers,
Rüdiger
